modulegen/internal/vscode/types.go

```diff
@@ -19,26 +20,26 @@ func newConfig(examples []string, modules []string) *Config {
 	config := Config{
 		Folders: []Folder{
 			{
-				Path: "../modulegen",
+				Path: filepath.Join("..", "modulegen"),
```

Comment:
suggestion: I really wouldn't make these changes, all OSs handle / separated relative paths and it just makes it harder to grok.